design/admin/templates/ezinfo/about.tpl

```diff
@@ -13,12 +13,14 @@
 
 <h3>{'What is eZ Publish?'|i18n( 'design/admin/ezinfo/about' )}</h3>
 {if is_set( $what_is_ez_publish )}
-    <p>{$what_is_ez_publish}</p>
+    {$what_is_ez_publish}
 {/if}
 
 <h3>{'License'|i18n( 'design/admin/ezinfo/about' )}</h3>
-{if is_set( $license )}
-    <p>{$license}</p>
+{if $license}
+    <iframe width="600" height="300" sandbox srcdoc="<html><body>{$license}</body></html>"></iframe>
```

Comment:
Right, looks like pre overflow:auto should work, then we also avoid the hackish html formatting.